phpstan.neon

```diff
@@ -4,6 +4,8 @@ parameters:
     ignoreErrors:
         - '#Function wincache_ucache_[a-zA-Z0-9_]+ not found#'
         - '#Function xcache_[a-zA-Z0-9_]+ not found#'
+        - '#Function mcrypt_[a-zA-Z0-9_]+ not found#'
```

Comment:
You need to revert this extra ignores you added. They are causing the travis build to fail since these methods are actually available there.